tests/perf/actor_id_scale/test.js

```diff
@@ -0,0 +1,55 @@
+import http from "k6/http";
+import { check } from "k6";
+import exec from "k6/execution";
```

Comment:
nit: why the inconsistency?

Reply:
That's how ESM imports work.

`import { check } from 'k6'` -> dereferences the object exported by k6 (so k6 probably has many things like `export function check()...` and other exports)

`import exec from k6/execution` -> imports the default export from the file.

Reply:
I had to check [this on MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import). This PR has been a series of TIL.